### DISCUSSION TITLE
Issue with Preview Deployment on Turborepo + pnpm Setup

### DISCUSSION THREAD

**Discussion by poteboy** [vercel/community#3517] [Help]
### Summary

Hello,

I've set up a project using Turborepo and pnpm, and I've included a website directory for the site. Almost 100% of the time, the Vercel preview deployment for this site fails with the following error:
```sh
Error: The file "/vercel/path0/website/.next/routes-manifest.json" couldn't be found. 
This is often caused by a misconfiguration in your project.
```

However, when I redeploy from the Vercel dashboard, it usually works without issues.

Any guidance on this would be much appreciated. Thank you!

### Example

https://vercel.com/musclenote/kuma-ui-website

### Steps to Reproduce

To reproduce this problem, you can create a PR on this repository: https://github.com/kuma-ui/kuma-ui.

**Comment by JaleelB** [comment]
Currently facing this same issue. Redeploying the failed deployment results in it deploying successfully but that isn't much of a fix
